docs/docs/end-users/Configuration.fsx

```diff
@@ -0,0 +1,170 @@
+(**
+---
+category: End-users
+categoryindex: 1
+index: 3
+---
+*)
+#r "nuget: Fantomas.Core, 5.0.0-beta-*"
+
+open Fantomas.Core.FormatConfig
+open Fantomas.Core
+let formatCode input configIndent =
+    CodeFormatter.FormatDocumentAsync  (false, input, configIndent)
+    |>  Async.RunSynchronously
+
+(**
+# Configuration
+Fantomas ships with a limited series of options.
+These can be stored in an [.editorconfig](https://editorconfig.org/) file and will be picked up automatically by the 
+commandline.   
+Your IDE should respect your settings, however the implementation of that is editor specific. Setting the configuration via 
+UI might be available depending on the IDE.
+*)
+
+(**
+## Usage
+Inside .editorconfig you can specify the file extension and code location to be use per config:
+```
+[*.fs]
+fsharp_space_before_uppercase_invocation = true
+
+# Write a comment by starting the line with a '#'
+[*.{fs,fsx,fsi}]
+fsharp_bar_before_discriminated_union_declaration = true
+
+# Apply specific settings for a targeted subfolder
+[src/Elmish/View.fs]
+fsharp_multiline_block_brackets_on_same_column = true
+fsharp_experimental_stroustrup_style = true
+```
+*)
+
+(**
+## Trying your settings via the online tool
+You can quickly try your settings via the <a href="https://fsprojects.github.io/fantomas-tools/#/fantomas/preview" target="_blank">online tool</a>.  
+<img src="{{root}}/online_tool_usage.gif" alt="drawing" width="100%"/>
+*)
+
+(**
+## Settings recommendations
+Fantomas ships with a series of settings that you can use freely depending  on your case. However, there are settings that we do not recommend and generally should not be used.   
+<i class="bi bi-check-circle-fill green-recommendation" ></i> <strong>Safe to change:</strong> Settings that aren't attached to any guidelines. Depending on your team or your own preferences, feel free to use these as it's been agreed on the codebase, however, you can always use it's defaults.     
```

Comment:
`feel free to use` => `feel free to change`, in a way they always are using the setting, even if it has a default value.